node/index.d.ts

```diff
@@ -396,6 +396,7 @@ declare namespace NodeJS {
         platform: string;
         mainModule?: NodeModule;
         memoryUsage(): MemoryUsage;
+        cpuUsage(previousValue?: [number, number]): [number, number];
```

Comment:
from the documentation, this function returns an object and not a tuple. it should be defined as 
```ts
cpuUsage(previousValue?: { user: number, system: number }): { user: number, system: number };
```